patches/coreboot-4.8.1/0009-Add-heads-TPM-measurements-to-Skylake-Kabylake.patch

```diff
@@ -0,0 +1,181 @@
+diff --git ./src/arch/x86/acpi.c ./src/arch/x86/acpi.c
+index 8b6b2c1..fca4a76 100644
+--- ./src/arch/x86/acpi.c
++++ ./src/arch/x86/acpi.c
+@@ -48,6 +48,7 @@
+ #include <cpu/x86/lapic_def.h>
+ #include <cpu/cpu.h>
+ #include <cbfs.h>
++#include <security/tpm/tss.h>
+ 
+ u8 acpi_checksum(u8 *table, u32 length)
+ {
+@@ -1020,6 +1021,10 @@ unsigned long write_acpi_tables(unsigned long start)
+ 		return current;
+ 	}
+ 
++	if (IS_ENABLED(CONFIG_MEASURED_BOOT)) {
++	  tlcl_measure(2, (const void*) dsdt_file, dsdt_size);
++	}
```

Comment:
Why do this? the dsdt_file was just loaded using `cbfs_boot_map_with_leak()` which itself calls the prog_segment_loaded so it's already been measured.

Reply:
Yup - my mistake. I willre move.